test/5.undici.test.js

```diff
@@ -45,6 +49,10 @@ describe('undici', () => {
       res.setHeader('x-agent', 'fast-proxy')
       res.send()
     })
+    service.get('/service/timeout', async (req, res) => {
+      await sleep(2000)
```

Comment:
why do we need 2 seconds of wait? I would prefer not to increase the slow down the testing.

Reply:
resolved - reduced to 200ms